### DISCUSSION TITLE
GS-1 QR Code with AI Application Identifier esp. FNC1

### DISCUSSION THREAD

**Discussion by msWP** [Shane32/QRCoder#483] [General]
Is it possible to crate a qr Code with FNC1 Type to build it as a QR CODE GS-1 Type?

**Comment by chavito706** [comment]
[](url)

**Comment by chavito706** [comment]
https://github.com/codebude/QRCoder/discussions/483#discussion-6090034

**Comment by codebude** [comment]
Yes and no. If you create the GS1 compatible payload as string and pass it to QRCoder it will generate a valid QR code. But no, there not payload generator for it at the moment. So there's no assistance in building the GS1 specific payload string.
